specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2022-09-08-preview/managedccf.json

```diff
@@ -0,0 +1,555 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-09-08-preview",
+    "title": "ConfidentialLedger",
+    "description": "Microsoft Azure Confidential Compute Ledger Managed CCF Control Plane REST API version 2022-09-08-preview."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ConfidentialLedger/managedCCF/{appName}": {
+      "get": {
+        "tags": [
+          "Managed CCF"
+        ],
+        "summary": "Retrieves information about a Managed CCF resource.",
+        "description": "Retrieves the properties of a Managed CCF app.",
+        "operationId": "Mccf_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedCCFNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; response contains the Managed CCF",
+            "schema": {
+              "$ref": "#/definitions/ManagedCCF"
+            }
+          },
+          "default": {
+            "description": "Error response describing the reason for operation failure. 404 - NotFound (Managed CCF does not exist.)",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ManagedCCFGet": {
+            "$ref": "./examples/Mccf_Get.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Managed CCF"
+        ],
+        "summary": "Deletes a Managed CCF resource.",
+        "description": "Deletes an existing Managed CCF.",
+        "operationId": "Mccf_Delete",
+        "x-ms-long-running-operation": true,
+        "parameters": [
```

Comment:
>parameters [](http://example.com/codeflow?start=9&length=10)

Use parameters from v3 common types for common parameters.

Reply:
thank you, i've applied v3 everywhere